client/ionic/src/pages/Triage.tsx

```diff
@@ -5,6 +5,7 @@ import * as Survey from 'survey-react';
 import 'survey-react/survey.css';
 import triage from '../surveys/triage.json';
 import 'tachyons';
+import '../theme/triage.css';
```

Comment:
While the css in here will not clash with other css, it will still be global.
You can use css modules to have it create a create a dynamic class.
This guide will work as is for our current app: https://dev.to/singhanuj620/using-css-module-in-latest-react-16-12-0-5e86